src/pycountry/db.py

```diff
@@ -119,15 +119,15 @@ def get(self, **kw):
     @lazy_load
     def lookup(self, value):
         # try relatively quick exact matches first
-        if isinstance(value, (str, unicode)):
-            value = value.lower()
```

Comment:
While this PR would definitely improve things, I would probably leave this in and use `value.upper()` since at least the alpha_2 indices are all in uppercase for me. I do think that this should probably happen on DB load and then any access should normalize the input to the case that's expected.

Reply:
Fair point, I opted for as small change as possible to not introduce side effects and IIRC looking at the commit history it looked like a simple oversight to lowercase after the lookup instead of before

Let me know what you think, I can merge as is but unfortunately wont have much time to spend on this PR at the moment as I have moved on to other projects.

Reply:
Well it looks like it's been awhile since this PR has been out and I'm just a friendly commenter who came to the same conclusion you did when looking at integrating this library into a project. I might wait to see if the author is interested in the change as is or if they wanted to overhaul things more (I was about to do that and submit a PR when I started looking at the open issues/PRs)

Reply:
Thanks for having that conversation and thanks for the suggestion. I have reviewed the code and your PR and decided to move forward (hopefully solving the problem you wanted to get solved) in a (only very slightly different) way. Check out cf66870180177b1709eea6aea97f7e5c562c6108 - and hopefully that aligns with your intentions.

Reply:
@ctheune looks good to me! I decided to pull in the package, and I'll see if there's anything else I was planning on suggesting as a PR, but it's nice to see that the project is still active :)